bal_tools/subgraph.py

```diff
@@ -37,16 +45,21 @@ def get_subgraph_url(self, subgraph="core") -> str:
         - https url of the subgraph
         """
         chain = "gnosis-chain" if self.chain == "gnosis" else self.chain
-
+        
         if subgraph == "core":
+            # UI has outdated core subgraph for polygon
+            if chain == "polygon":
+                return "https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-polygon-v2"
             magic_word = "subgraph:"
         elif subgraph == "gauges":
             magic_word = "gauge:"
         elif subgraph == "blocks":
             magic_word = "blocks:"
-            ## UI has no blocks subgraph for op
+            ## UI has no blocks subgraph for op or gnosis
             if chain == "optimism":
                 return "https://api.thegraph.com/subgraphs/name/iliaazhel/optimism-blocklytics"
+            elif chain == "gnosis-chain":
```

Comment:
A little unclear why you moved it and unsure of this ordering, can you explain?

Reply:
wdym ordering? i dont think it should matter here